README.md

```diff
@@ -84,6 +84,8 @@ Composer shipped as jar, to run it you need JVM 1.8+: `java -jar composer-latest
   * Key-value pairs to pass to Instrumentation Runner. Usage example: `--instrumentation-arguments myKey1 myValue1 myKey2 myValue2`.
 * `--verbose-output`
   * Either `true` or `false` to enable/disable verbose output for Composer. `false` by default.
+* `--keep-output-on-exit`
+  * Keep output on exit. False by default.
```

Comment:
Hm, I can see how someone can be confused by this and think that Composer has a way to drop output from console.

Maybe we should explicitly say something like "Keep intermediate output files on exit. `false` by default.`?

Also, looking at it again, I'd make it a boolean flag, so you could actually control it "programmatically" or through an env variable.

Boolean would also make it more consistent with `--verbose-output`